src/components/editor/editor-window/autocomplete/autocomplete.ts

```diff
@@ -16,13 +17,27 @@ export const emojiStrategy = (editor: Editor): StrategyProps => ({
     if (matches && matches.length >= 2) {
       term = matches[2].toLowerCase()
     }
-    callback(emojiIndex.search(term) || [])
+    callback(emojiIndex.search(term, { emojisToShowFilter: () => true, exclude: [], include: [], maxResults: 10, custom: customEmojis }) || [])
   },
   template: (value) => {
-    return `${(value as BaseEmoji).native} ${(value as BaseEmoji).name}`
+    let replacement = ''
+    if ((value as BaseEmoji).native) {
+      replacement = `${(value as BaseEmoji).native} ${(value as BaseEmoji).name}`
+    } else if ((value as CustomEmoji).imageUrl) {
+      // noinspection CheckTagEmptyBody
+      replacement = `<i class="fa ${(value as CustomEmoji).name}"></i> ${(value as BaseEmoji).name}`
+    }
+    return replacement
   },
   replace: (value) => {
-    return `$1${(value as BaseEmoji).colons}`
+    let replacement = ''
+    if ((value as BaseEmoji).native) {
+      replacement = `${(value as BaseEmoji).native}`
```

Comment:
Should autocomplete input the native emoji here instead of the shortcode variant (`colons`)?

Reply:
Nevermind, just saw #397